src/plugins/audio/audio.yml

```diff
@@ -7,7 +7,7 @@ source:
 externalLink:
   en: External link
   de: Externer Link
-elmuCdn:
+internalCdn:
   en: Elmu CDN
```

Comment:
We can call these "Internal CDN"/"Internes CDN"

Reply:
Ah, I thought we can override somehow the values in the future.

Reply:
Done